workflow-service/pom.xml

```diff
@@ -36,7 +36,11 @@
     </properties>
 
     <dependencies>
-        <!-- START Web Dependencies -->
+        <dependency>
+            <groupId>io.kubernetes</groupId>
```

Comment:
Why do we need this update here?

Reply:
we have a few dependencies that depend on this artifact. Each one wants a different version. The way to solve it is by declaring the dependency directly in the higher level pom so it forces a version for all of them.  Obviously, the highest version since it is backward compatible.

Reply:
you are right, I see issue with versions and we need to be explicit